test/general-tests/starknet-new-account/check.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -eu
```

Comment:
This test can assert that a file has indeed been created. Also, the existing tests which do `starknet-deploy-account` should be updated to create the account first (and probably fund it).

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/246/commits/4edf1fcaf412c6c0e64a6172c25b6d526cb9c334

Reply:
What about the existing tests using `hardhat starknet-deploy-account`, i.e. using the script `deploy-funded-cli-account.sh`